docs/configuration_and_usage.rst

```diff
@@ -0,0 +1,93 @@
+Configuration and usage
+=======================
+
+Getting around administrator interface
+--------------------------------------
+
+`Edx-enterprise` is integrated with edx-platform Django admin. It can be accessed at
+`$LMS_SERVER_ADDRESS/admin/enterprise` URL.
+
+.. image:: images/enterprise-admin.png
+
+So far, there are two items in the admin interface:
+
+* "Enterprise Customers" - lists and manages Enterprise Customer records.
+* "Enterprise Customer Users" - lists and manages enterprise customer students.
+
+.. image:: images/enterprise-customer.png
+
+Despite providing student management interface in Enterprise Customer Users section, `edx-enterprise` admin provides
+custom (and richer) interface for managing learners associated with particular Enterprise Customer in "Enterprise
+Customer" section, accessible via "Manage Learners" button in the top-right corner of Enterprise Customer edit
+interface.
+
+Manage Learners View
+--------------------
+
+Manage Learners view main goal is to provide means to:
```

Comment:
"The Manage Learners view is meant to be used by non-technical staff to administer enterprise learners. It can be used to:"